lib/membrane_http_adaptive_stream/manifest/track.ex

```diff
@@ -157,25 +172,47 @@ defmodule Membrane.HTTPAdaptiveStream.Manifest.Track do
      %__MODULE__{track | stale_segments: stale_segments, stale_headers: stale_headers}}
   end
 
-  @spec discontinue(t()) ::
-          {{header_name :: String.t(), discontinuity_seq_num :: non_neg_integer()}, t()}
+  def add_segment(%__MODULE__{finished?: true} = _track, _duration, _attributes),
+    do: raise("Cannot add new segments to finished track")
+
+  @doc """
+  Discontinue the track, indicating that parameters of the stream have changed.
+  New header has to be stored under the returned filename.
+  For details on discontinuity, please refer to RFC 8216.
```

Comment:
I would put some new lines in the doc itself so that `ex_doc` can distinguish between the necessary brief documentation part and the additional information like mentioning the RFC.